### ISSUE TITLE
Graylog extractor converter crashing on build

### ISSUE THREAD

**Issue by mitchtjohnston** [suzuki-shunsuke/go-graylog#187]
Hey!

I have run in to an issue deploying graylog after adding a converter to an extractor.

Current Input:
```
resource "graylog_extractor" "http_response_code" {
  input_id        = "${graylog_input.syslog_udp.id}"
  title           = "Apache http_response_code"
  type            = "regex"
  cursor_strategy = "copy"
  source_field    = "message"
  target_field    = "http_response_code"
  condition_type  = "regex"
  condition_value = "[1-5]\\d{2}"
  order           = 0
  
  converters {
    type = "numeric"
    config {
      date_format = ""
      time_zone = ""
      locale = ""
    }
  }

  regex_type_extractor_config = {
    regex_value = "HTTP/1.[0-1]\" (\\d{3}) "
  }
}
```
Originally I have closed off the `converters.config` empty as I was setting it as numeric:
```
converters {
    type = "numeric"
    config {}
  }
```
However the build failed with the following error: 
`Error: graylog_extractor.http_response_code: "converters.0.config.0.date_format": required field is not set`

Once adding the empty fields for `date_format, time_zone and locale` terraform crashed:
```
Error: Error applying plan:
--
  |  
  | 3 errors occurred:
  | * graylog_extractor.http_response_code: 1 error occurred:
  | * graylog_extractor.http_response_code: unexpected EOF
  |  
  |  
  | * graylog_extractor.user_domain (destroy): 1 error occurred:
  | * graylog_extractor.user_domain: unexpected EOF
  |  
  |  
  | * graylog_input.gelf_udp: 1 error occurred:
  | * graylog_input.gelf_udp: unexpected EOF
...
2019-10-24T03:00:32.420Z [DEBUG] plugin: plugin process exited: path=/home/agent/.terraform.d/plugins/terraform-provider-graylog_v8.1.0
--
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error occurred:
  | * graylog_extractor.http_response_code: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error occurred:
  | * graylog_extractor.http_response_code: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [TRACE] [walkApply] Exiting eval tree: graylog_extractor.http_response_code
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error occurred:
  | * graylog_extractor.user_domain: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error occurred:
  | * graylog_extractor.user_domain: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalOpFilter, err: 1 error occurred:
  | * graylog_extractor.user_domain: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [TRACE] [walkApply] Exiting eval tree: graylog_extractor.user_domain (destroy)
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalApplyPost, err: 1 error occurred:
  | * graylog_input.gelf_udp: unexpected EOF
  |  
  | 2019/10/24 03:00:32 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error occurred:
  | * graylog_input.gelf_udp: unexpected EOF
...
graylog_extractor.http_response_code: Creating...
--
  | condition_type:                            "" => "regex"
  | condition_value:                           "" => "[1-5]\\d{2}"
  | converters.#:                              "" => "1"
  | converters.0.config.#:                     "" => "1"
  | converters.0.type:                         "" => "numeric"
  | cursor_strategy:                           "" => "copy"
  | input_id:                                  "" => "***REDACTED-BY-SM-DEPLOY***"
  | order:                                     "" => "0"
  | regex_type_extractor_config.#:             "" => "1"
  | regex_type_extractor_config.0.regex_value: "" => "HTTP/1.[0-1]\" (\\d{3}) "
  | source_field:                              "" => "message"
  | target_field:                              "" => "http_response_code"
  | title:                                     "" => "Apache http_response_code"
  | type:                                      "" => "regex"
  | graylog_input.gelf_udp: Modifying... (ID: 5cff024b55369000105d4a74)
  | attributes.0.allow_override_date: "false" => "true"
  |  
  |  
  | !!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
  |  
  | Terraform crashed! This is always indicative of a bug within Terraform.
  | A crash log has been placed at "crash.log" relative to your current
  | working directory. It would be immensely helpful if you could please
  | report the crash with Terraform[1] so that we can fix this.
```

**Comment by suzuki-shunsuke** [comment]
Please tell us some information.

* Terraform version
* Graylog version
*terraform-provider-graylog version (please use latest)

**Comment by suzuki-shunsuke** [comment]
Currently, `converters[].config.date_format` is required but it's wrong. We should fix this.

* https://github.com/suzuki-shunsuke/go-graylog/blob/5316e401e1eb5691b6b5b51eac6f5986ce663898/terraform/docs/extractor.md#common-required-argument
* https://github.com/suzuki-shunsuke/go-graylog/blob/5316e401e1eb5691b6b5b51eac6f5986ce663898/terraform/graylog/resource_extractor.go#L177

## Sample request

POST /system/inputs/{inputId}/extractors

Request Body

```json
{
  "title": "Apache http_response_code",
  "extractor_type": "regex",
  "cut_or_copy": "copy",
  "source_field": "message",
  "target_field": "http_response_code",
  "condition_type": "regex",
  "condition_value": "[1-5]\\d{2}",
  "order": 0,
  "converters": {
    "numeric": {}
  },
  "extractor_config": {
    "regex_value": "HTTP/1.[0-1]\" (\\d{3}) "
  }
}
```

**Comment by suzuki-shunsuke** [comment]
I have created the pull request #189

**Comment by suzuki-shunsuke** [comment]
I have released [v8.1.1](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v8.1.1). Please check.

**Comment by suzuki-shunsuke** [comment]
I close this issue once.
If you have any opinions, please comment.
